src/components/pages/traits/edit/index.tsx

```diff
@@ -0,0 +1,593 @@
+            transform="translateY(-50%)"
+            cursor="grab"
+            zIndex={2}
+          >
+            <DragHandleIcon boxSize={5} color="gray.500" />
+          </Box>
+        )}
+      <TextBoxField
+        name={`translations[${translationSelected}].values[${value}].value`}
+        label={
+          hForm.watch(`translations`).filter((t) => t.traits.languageId == languageId)[0].values[
+            value
+          ].value &&
+          hForm.watch(`translations`)[translationSelected].traits.languageId != languageId
+            ? hForm.watch(`translations`).filter((t) => t.traits.languageId == languageId)[0]
+                .values[value].value
+            : t("traits:create_form.value")
+        }
+      />
+      <TextBoxField
+        name={`translations[${translationSelected}].values[${value}].description`}
+        label={`${t("traits:create_form.description")} ${value + 1}`}
+      />
+      <Box ml={4}>
+        <Button
+          type="button"
+          as="label"
+          cursor="pointer"
+          w="15"
+          size={"sm"}
+          htmlFor={value}
+          height={70}
+          width={70}
+          border={"2px dashed #aaa"}
+        >
+          {hForm.watch(`translations[${translationSelected}].values`)[value].icon && (
+            <Image
+              boxSize="2.2rem"
+              objectFit="contain"
+              src={getTraitIcon(
+                hForm.watch(`translations[${translationSelected}].values`)[value].icon
+              )}
+              alt={hForm.watch(`translations[${translationSelected}].values`)[value].icon}
+              ignoreFallback={true}
+              mb={2}
+            />
+          )}
+          <VisuallyHidden
+            as="input"
+            type="file"
+            id={value}
+            accept="image/*"
+            onChange={handleOnPhotoUpload}
+          />
+        </Button>
+      </Box>
+      <Box display="flex" justifyContent="center" alignItems="center">
+        {!hForm.watch(`translations[${translationSelected}].values`)[value].id &&
+          hForm.watch(`translations`)[translationSelected].traits.languageId == languageId && (
+            <Button
+              aria-label="Remove value"
+              onClick={(e) => {
+                e.stopPropagation(); // Prevent drag event
+                removeValue(value); // Call delete function
+              }}
+              size="sm"
+              colorScheme="red"
+            >
+              <CloseIcon />
+            </Button>
+          )}
+      </Box>
+    </SimpleGrid>
+  ));
+
+  const SortableList = SortableContainer<{ items: any }>(({ items }) => {
+    return (
+      <Box>
+        {items.map((value, index) => (
+          <SortableItem key={`item-${value.id}`} index={index} value={index} />
+        ))}
+      </Box>
+    );
+  });
+
+  const shouldCancelStart = (e) => {
+    // Ignore clicks on file inputs or labels
+    return (
+      e.target.tagName === "INPUT" ||
```

Comment:
have all these hardcoded values in the constansts.ts file and then refer them here. Make such changes in every place applicable in your code